.circleci/config.yml

```diff
@@ -1386,6 +1387,175 @@ jobs:
                 --branch=all --branch-limit=5 --limit=2 \
                 --email=cloudtech-builds@scalyr.com
 
+  build-windows-package:
+    description: "Build windows MSI installer."
+    executor:
+      name: win/default
+      size: "medium"
+      shell: powershell.exe
+    steps:
+      - checkout
+      - restore_cache:
+          key: windows_python_installer-python-3.7.7-{{ .Branch }}
+
+      - run:
+          name: Install python.
+          command: |
+            $ProgressPreference = "SilentlyContinue"
+            mkdir python_installer -Force
+            if (!(Test-Path .\python_installer\python-3.7.7-amd64.exe -PathType Leaf)) {
+                wget -O .\python_installer\python-3.7.7-amd64.exe https://www.python.org/ftp/python/3.7.7/python-3.7.7-amd64.exe
+            }
+            .\python_installer\python-3.7.7-amd64.exe /install /quiet TargetDir=c:\Python37 Include_tcltk=0
+
+      - save_cache:
+          key: windows_python_installer-{{ .Branch }}
+          paths:
+            - "python_installer"
+
+      - run:
+          name: build MSI package.
+          command: |
+             .\win32\build.ps1 c:\Python37
+              mkdir package
+              mv .\ScalyrAgentInstaller-*.msi package\ScalyrAgentInstaller.msi
+
+      - persist_to_workspace:
+          root: package
+          paths: ScalyrAgentInstaller.msi
+
+      - store_artifacts:
+          path: package
+
+  build-linux-packages:
```

Comment:
Now that we have this job, we can probably get rid of ``tests/distribution/{rpm_deb}_package.py`` file, right?

Reply:
Yes. I think new build job does the same thing as `tests/distribution/{rpm_deb}_package`